ansible/postdeploy.yml

```diff
@@ -6,4 +6,8 @@
 - hosts: ansible
   tasks:
     - include: tasks/installCatalog.yml
-      when: mode == "deploy"
\ No newline at end of file
+      when: mode == "deploy"
+      
```

Comment:
It looks like the catalog is being installed from both openwhisk and openwhisk-catalog.  Is this by design?  Currently, these look identical so we won't really know if they are being installed properly from openwhisk-catalog if they had just been installed by openwhisk.  Seems to defeat the purpose of verifying that the new install is working properly.